unittest/test_idmap.cpp

```diff
@@ -286,13 +288,13 @@ TEST_P(IDMAPTest, idmap_copy) {
 
     // cpu to gpu
     ASSERT_ANY_THROW(knowhere::cloner::CopyCpuToGpu(index_, -1, conf));
-    auto clone_index = knowhere::cloner::CopyCpuToGpu(index_, DEVICEID, conf);
+    auto clone_index = knowhere::cloner::CopyCpuToGpu(index_, DEVICE_ID, conf);
     auto clone_result = clone_index->Query(query_dataset, conf, nullptr);
 
     AssertAnns(clone_result, nq, k);
     ASSERT_THROW({ std::static_pointer_cast<knowhere::GPUIDMAP>(clone_index)->GetRawVectors(); },
                  knowhere::KnowhereException);
-    ASSERT_ANY_THROW(clone_index->Serialize(conf));
+    ASSERT_NO_THROW(clone_index->Serialize(conf));
```

Comment:
you can comment this line, don't change it.
We expect some exception throw out, if it does not throw exception any more, we need check what's changed.